src/app.js

```diff
@@ -2,48 +2,36 @@ const { app } = require("./support/setupExpress");
 const { query } = require("./support/db");
 const { gameOfThronesEpisodes } = require("./data/gameOfThronesData");
 
-/** 
- @typedef {import('./data/episodeType').Episode} Episode
-*/
 
-//You can delete this once you see the episodes have loaded ok.
-summariseEpisodesToConsole(gameOfThronesEpisodes);
 
-//configure the server's route handlers
 app.get("/", (req, res) => {
-    res.render("pages/index");
+    res.render("pages/index", { 
+        
+    });
```

Comment:
it'd be better to remove this empty object passed to the EJS.  though i get it if you're leaving it as a place-holder to remember it's even possible.